Makefile-libostree.am

```diff
@@ -205,7 +205,8 @@ gpgreadmedir = $(pkgdatadir)/trusted.gpg.d
 EXTRA_DIST += src/libostree/README-gpg src/libostree/bupsplit.h \
 		src/libostree/ostree-enumtypes.h.template \
 		src/libostree/ostree-enumtypes.c.template \
-		src/libostree/ostree-deployment-private.h
+		src/libostree/ostree-deployment-private.h \
+		src/libostree/ostree-repo-deprecated.h
```

Comment:
This also needs to go in `Makefile-libostree-defines.am` in order to be installed.